docs/indexing_samples.md

```diff
@@ -0,0 +1,51 @@
+# Adding sample indexing to the index runner
+
+## Reusable code
+
+* The event loop in `main.py`
+* The logging code in `main.py`
+* `config.py` (currently some code in `main.py` could probably be moved into `config.py` for
+  separation of concerns reasons)
+* The Kafka code in `main.py` with configurable topics
+
+## Non-reusable code
+
+* `_handle_message` in `main.py` and the methods it calls
+
+## Refactoring to avoid (nearly) duplicate code for sample indexing
+
+* Refactor the event loop so that the message handling function can be passed into the event loop
```

Comment:
Maybe this would be good, but I don't understand what you're trying to do. Why can't the sample indexer be a module inside `index_runner/es_indexers`?

Reply:
We discussed this OOB, but the sample service has it's own topic, and (probably) needs its own event handler.